pinot-core/src/test/java/com/linkedin/pinot/queries/InterSegmentAggregationMultiValueQueriesTest.java

```diff
@@ -0,0 +1,334 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.queries;
+
+import com.linkedin.pinot.common.response.broker.BrokerResponseNative;
+import org.testng.annotations.Test;
+
+
+public class InterSegmentAggregationMultiValueQueriesTest extends BaseMultiValueQueriesTest {
+  private static String SV_GROUP_BY = " group by column8";
+  private static String MV_GROUP_BY = " group by column7";
+
+  @Test
+  public void testCountMV() {
+    String query = "SELECT COUNTMV(column6) FROM testTable";
+
+    BrokerResponseNative brokerResponse = getBrokerResponseForQuery(query);
+    QueriesTestUtils.verifyAggregationResult(brokerResponse, 400000L, 0L, 400000L, 400000L, new String[]{"426752"});
+
+    brokerResponse = getBrokerResponseForQueryWithFilter(query);
+    QueriesTestUtils.verifyAggregationResult(brokerResponse, 62480L, 1129720L, 62480L, 400000L, new String[]{"62480"});
+
+    brokerResponse = getBrokerResponseForQuery(query + SV_GROUP_BY);
+    QueriesTestUtils.verifyAggregationResult(brokerResponse, 400000L, 0L, 800000L, 400000L, new String[]{"231056"});
+
+    brokerResponse = getBrokerResponseForQuery(query + MV_GROUP_BY);
+    QueriesTestUtils.verifyAggregationResult(brokerResponse, 400000L, 0L, 800000L, 400000L, new String[]{"199896"});
+  }
+
+  @Test
+  public void testMaxMV() {
+    String query = "SELECT MAXMV(column6) FROM testTable";
+
+    BrokerResponseNative brokerResponse = getBrokerResponseForQuery(query);
+    QueriesTestUtils.verifyAggregationResult(brokerResponse, 400000L, 0L, 400000L, 400000L,
```

Comment:
I am not a big fan of having these hard coded value expected values, because if we want to add a new test, we don't know what the correct answer is. I guess you ran it once to get the answer, that defeats the purpose of the test.